OnlineFlightSearchAPI/Models/FlightDetail.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace OnlineFlightSearchAPI.Models
+{
+    public class FlightDetail
+    {
+        public FlightDetail(string flightCode, string startLocation, string destination, string departureDate, string travelTime, decimal ticketFare, int availabilityCount)
+        {
+            FlightCode = flightCode;
+            StartLocation = startLocation;
+            Destination = destination;
+            DepartureDate = Convert.ToDateTime(departureDate);
+            TravelTime= travelTime;
+            TicketFare = ticketFare;
+            AvailabilityCount = availabilityCount;
+        }
+
+        public string FlightCode { get; set; }
+
+        public string StartLocation { get; set; }
+
+        public string Destination { get; set; }
+
+        public DateTime DepartureDate { get; set; }
+
+        public string TravelTime { get; set; }
+
+        public decimal TicketFare { get; set; }
```

Comment:
do you really need a decimal?

Reply:
Ticket Fare Type may have decimal values. so kept like that. If needed can change it back to float/int.

Reply:
do you really such long number? double would be suffice for most cases

Reply:
Since it is Currency Type. Kept as Decimal to allow maximum precision after rounding off